app/src/main/java/fr/free/nrw/commons/wikidata/WikidataEditService.kt

```diff
@@ -79,6 +79,7 @@ class WikidataEditService @Inject constructor(
         fileEntityId: String?,
         depictedItems: List<String>
     ): Observable<Boolean> {
+
```

Comment:
Please perform such stylistic changes in a separate style-only pull request. Thanks!